sniffles/ruletrafficgenerator.py

```diff
@@ -1600,95 +1642,107 @@ def generate_nfa_data(self, rule=None):
     def generate_http_content(self, rules):
         http_directive_map = {}
         # HTTP/1.1
-        http_text = [72, 84, 84, 80, 47, 49, 46, 49]
+        http_text = [72, 84, 84, 80, 47, 49, 46, 49, 13, 10]
```

Comment:
Adding cr lf here might break elsewhere.  The current format is supposed to add that separate.  However, that may be an error below.